plugins/modules/database/postgresql/postgresql_query.py

```diff
@@ -175,23 +175,45 @@
 
 RETURN = r'''
 query:
-    description: Query that was tried to be executed.
+    description:
+    - Executed query.
+    - When reading several queries from a file, it contains the last one.
     returned: always
     type: str
     sample: 'SELECT * FROM bar'
 statusmessage:
-    description: Attribute containing the message returned by the command.
+    description:
+    - Attribute containing the message returned by the command.
+    - When reading several queries from a file, it contains a message of the last one.
     returned: always
     type: str
     sample: 'INSERT 0 1'
 query_result:
     description:
     - List of dictionaries in column:value form representing returned rows.
-    returned: changed
+    - When running queries from a file, returns result of the last query.
+    returned: always
     type: list
+    elements: dict
     sample: [{"Column": "Value1"},{"Column": "Value2"}]
+query_list:
+    description:
+    - List of executed queries.
+      Useful when reading several queries from a file.
+    returned: always
+    type: list
+    elements: str
+    sample: ['SELECT * FROM foo', 'SELECT * FROM bar']
+query_all_results:
+    description:
+    - List containing results of all queries executed (one sublist for every query).
+      Useful when reading several queries from a file.
+    returned: always
+    type: list
+    elements: list
+    sample: [[{"Column": "Value1"},{"Column": "Value2"}], [{"Column": "Value1"},{"Column": "Value2"}]]
 rowcount:
-    description: Number of affected rows.
+    description: Total number of produced or affected rows.
```

Comment:
When using a script with multiple queries, it contains an all affected rows. But this is not clear from the description. Please describe it more precise.

Reply:
Agreed, needs clarification, thank! fixed